File: evidence/generated-systems/golang/healthcare-on-golang/healthcare-on-golang/internal/model/Allergy.go
package model

import (
    "gorm.io/gorm"
)

//==============================================================
// Allergy Declaration
//==============================================================
type Allergy struct {
    gorm.Model
     Substance                                    string
    Reaction                                    string
    PatientId         *uint
    Patient           *Patient `gorm:"foreignKey:PatientId"`
    Severity                      AllergySeverity
    Status                      AllergyStatus

// parent associations as their child

}

File: evidence/generated-systems/angular/healthcare-on-angular/src/app/models/Allergy.js

// Define collection and schema for Allergy
export interface Allergy {
    substance:
	type : string
    reaction:
	type : string
    Patient:
	type : Schema.Types.ObjectId
    Severity:
 	type : String
    Status:
 	type : String
#
    collection: 'allergys'
}
